charts/sequencer/files/cometbft/config/genesis.json

```diff
@@ -1,42 +1,42 @@
 {
   "app_hash": "",
   "app_state": {
-    "native_asset_base_denomination": "{{ .Values.genesis.nativeAssetBaseDenomination }}",
+    "nativeAssetBaseDenomination": "{{ .Values.genesis.nativeAssetBaseDenomination }}",
     "fees": {
-      "transfer_base_fee": {{ .Values.genesis.fees.transferBaseFee }},
-      "sequence_base_fee": {{ .Values.genesis.fees.sequenceBaseFee }},
-      "sequence_byte_cost_multiplier": {{ .Values.genesis.fees.sequenceByteCostMultiplier }},
-      "init_bridge_account_base_fee": {{ .Values.genesis.fees.initBridgeAccountBaseFee }},
-      "bridge_lock_byte_cost_multiplier": {{ .Values.genesis.fees.bridgeLockByteCostMultiplier }},
-      "bridge_sudo_change_fee": {{ .Values.genesis.fees.bridgeSudoChangeFee }},
-      "ics20_withdrawal_base_fee": {{ .Values.genesis.fees.ics20WithdrawalBaseFee }}
+      "transferBaseFee": {{ include "sequencer.fee" .Values.genesis.fees.transferBaseFee }},
+      "sequenceBaseFee": {{ include "sequencer.fee" .Values.genesis.fees.sequenceBaseFee }},
+      "sequenceByteCostMultiplier": {{ include "sequencer.fee" .Values.genesis.fees.sequenceByteCostMultiplier }},
+      "initBridgeAccountBaseFee": {{ include "sequencer.fee" .Values.genesis.fees.initBridgeAccountBaseFee }},
+      "bridgeLockByteCostMultiplier": {{ include "sequencer.fee" .Values.genesis.fees.bridgeLockByteCostMultiplier }},
+      "bridgeSudoChangeFee": {{ include "sequencer.fee" .Values.genesis.fees.bridgeSudoChangeFee }},
+      "ics20WithdrawalBaseFee": {{ include "sequencer.fee" .Values.genesis.fees.ics20WithdrawalBaseFee }}
     },
-    "allowed_fee_assets": [
+    "allowedFeeAssets": [
       {{- range $index, $value := .Values.genesis.allowedFeeAssets }}
       {{- if $index }},{{- end }}
       "{{ $value }}"
       {{- end }}
     ],
-    "ibc_params": {
-      "ibc_enabled": {{ .Values.genesis.ibc.enabled }},
-      "inbound_ics20_transfers_enabled": {{ .Values.genesis.ibc.inboundEnabled }},
-      "outbound_ics20_transfers_enabled": {{ .Values.genesis.ibc.outboundEnabled }}
+    "ibcParameters": {
+      "ibcEnabled": {{ .Values.genesis.ibc.enabled }},
+      "inboundIcs20TransfersEnabled": {{ .Values.genesis.ibc.inboundEnabled }},
+      "outboundIcs20TransfersEnabled": {{ .Values.genesis.ibc.outboundEnabled }}
     },
-    "address_prefixes": {
+    "addressPrefixes": {
       "base": "{{ .Values.genesis.addressPrefixes.base }}"
     },
     "accounts": [
       {{- range $index, $value := .Values.genesis.genesisAccounts }}
       {{- if $index }},{{- end }}
       {
         "address": {{ include "sequencer.address" $value.address }},
-        "balance": {{ toString $value.balance | replace "\"" "" }}
+        "balance": {{ include "sequencer.fee" ( toString $value.balance | replace "\"" "" ) }} 
```

Comment:
This change disallows some hight initial balances that were previously possible. Limited to uint64 balances effectively.

Reply:
Yes indeed. For our current use cases this is fine. A followup should fix this. I wonder if there is a way to call out to a helper script using helm?

Reply:
We could have an init script which does the generation.

Reply:
I have created https://github.com/astriaorg/astria/issues/1388 as a followup.